tests/sparseml/pytorch/image_classification/test_export.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import onnx
+import pytest
+import torch
+
+from click.testing import CliRunner
+from sparseml.pytorch.image_classification.export import main
+from sparseml.pytorch.models import resnet50
+
+
+@pytest.fixture()
+def temp_dir(tmp_path):
+    """
+    return: a temporary directory
+    """
+    test_dir = tmp_path / "test_dir"
+    test_dir.mkdir()
+    yield test_dir
+
+
+@pytest.fixture()
+def non_quant_resnet_checkpoint(temp_dir):
+    """
+    return: path to a ResNet50 checkpoint
+    """
+    checkpoint_path = os.path.join(temp_dir, "model.pth")
+    model = resnet50()
+    torch.save(model, checkpoint_path)
+    yield checkpoint_path
+
+
+@pytest.fixture()
+def quant_recipe_path():
+    """
+    return: path to a resnet50 recipe with quantization
+    """
+    recipe_path = "./resnet_test_recipe.md"
+    recipe_path = os.path.abspath(recipe_path)
+    yield recipe_path
+
+
+def test_export_one_shot(non_quant_resnet_checkpoint, quant_recipe_path, temp_dir):
```

Comment:
we should be checking pruning here as well, especially since you're using a pruning recipe. (use the `tensor_sparsity` helper, see comments in the recipe template PR)

Reply:
Tensor sparsity takes in a torch tensor, I used ModelAnalysis tool in upcoming commit to assert sparsity